### PR TITLE
Changed trees

### PR DESCRIPTION
**Pull request by Zlata-K**

### CODE REVIEW

**Comment by ClaudiaLapalme** [comment]
Can you pull the latest version of master in your branch just to make sure there's no merge conflicts occurring

File: Assets/Prefabs/Tiles/Fillers/Tree Filler.prefab [hunk 1]
```diff
@@ -29,15 +29,15 @@ PrefabInstance:
       objectReference: {fileID: 0}
     - target: {fileID: 4791311736922912, guid: fa7edeb6b77fa4a48bd89ca8b69c8bb7, type: 3}
       propertyPath: m_LocalScale.x
-      value: 0.5
+      value: 0.51
       objectReference: {fileID: 0}
     - target: {fileID: 4791311736922912, guid: fa7edeb6b77fa4a48bd89ca8b69c8bb7, type: 3}
       propertyPath: m_LocalScale.y
-      value: 0.5
+      value: 0.51
```
Comment (f-michaut) [inline Assets/Prefabs/Tiles/Fillers/Tree Filler.prefab:78]: Why did you changed the scale of one of the trees ?

**Review by f-michaut — CHANGES_REQUESTED** [review]
There is a mesh collider on all the trees inside the TreeFiller prefab, but the collider is missing the `mesh` parameter to work, can you fix it ?

There is already a mesh on the `old tree`, `tree_full_with_leaves` and `acacia\default` objects, so i don't know if it's better to add the mesh collider on them or to add a mesh renderer on the Tree1 / Tree2 / Tree3 directly it's up to you !

File: Assets/Prefabs/Tiles/Fillers/Tree Filler.prefab (hunk 1 above)
Comment (Zlata-K) [reply Assets/Prefabs/Tiles/Fillers/Tree Filler.prefab:78]: Whoops, by accident 😅

**Comment by Zlata-K** [comment]
Fixed

**Review by tedgag — APPROVED** [review]
Looks nice! Approved

**Review by f-michaut — CHANGES_REQUESTED** [review]
Okay looks good ! 

Altho, I can only see 2 trees (i don't see the acacia one) can you check that ?
And there is still the BoxColliders on the Tree1, can you remove them ?

**Review by f-michaut — APPROVED** [review]

**Pull request merged**